server/ott-config.ts

```diff
@@ -554,23 +559,29 @@ function postProcessConfig(): void {
 	}
 }
 
+function validateConf(): Result<void, Error> {
+	try {
+		conf.validate({ allowed: "strict" });
+		return ok(undefined);
+	} catch (e) {
+		return err(e);
+	}
+}
```

Comment:
Use `intoResult()` for this.

Reply:
Trying to do so now but having some trouble. I see how the two are super similar where `conf.validate()` lines up with the result in `intoResult()`

I'm trying to do `intoResult(conf.validate({allowed: "strict"})`, but it isn't working since I think the `func()` in `intoResult()` takes no args.

Hint or maybe some clarification on if I'm right?

Reply:
`intoResult` takes a function. `intoResult(() => foo())`

Reply:
Why doesn't something like this work?

```
function validateConf() {
	conf.validate({ allowed: "strict" });
}

const confResult = intoResult(validateConf());
```

Reply:
Nevermind, it does. Dumb mistake.